Makefile

```diff
@@ -30,69 +30,69 @@ else
 GOBIN=$(shell go env GOBIN)
```

Comment:
note this will be removed once https://github.com/DataDog/datadog-operator/pull/166 gets merged